src/screens/Notifications/AddNotificationEmailScreen.tsx

```diff
@@ -46,8 +46,18 @@ class AddNotificationEmailScreen extends PureComponent<Props, State> {
     email: '',
   };
 
+  forceUpdateComponent = () =>
```

Comment:
name is misleading, it suggests the immediate action of updating component after calling it. I would add something like `addListenerResetStateOnNavigationFocus`

Reply:
hmm true but prefer `handlerResetStateOnNavigationFocus`

Reply:
up